### ISSUE TITLE
Emoji Search

### ISSUE THREAD

**Issue by anthonybench** [raycast/extensions#12406]
### Extension

https://github.com/raycast/extensions/tree/1602734ca5c073d40aaaf0557da951d871f71afa/extensions/emoji/

### Description

Right now in the emoji search settings, you can choose to either copy to clipboard, or paste in active app under the `Primary Action` setting.

I want both.

I'm sure the maintainer/developer would know the most elegant way to go about this, and I don't much care how it looks, whether it be a multiselect list, or checkboxes or what have you, but it would be nice to hit enter with an emoji selected, then if i want multiples of that emoji (very common case), i can ⌘V to add additional copies of the emoji since it would *also* already be in my clipboard.

TLDR; please let me choose to **both** put the emoji in my clipboard **and** paste the emoji into the active app, instead of choosing between one or the other. 🙇🙇🙇

### Who will benefit from this feature?

Everyone! Wanting multiples of an emoji is incredibly common 🎉

### Anything else?

Thank you so much for your time, this extension is better than any other emoji tool out there! I'm a huge fan!
